### PR TITLE
Es refactor

### PR DESCRIPTION
**Pull request by KagamiChan**
refactor in es, reduxify

### CODE REVIEW

File: index.es [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+    nextHour.setMinutes (0)
+    nextHour.setSeconds (0)
+    nextHour.setMilliseconds (0)
+    scheduler.schedule (this.hourly_notify,
+      {
+        time: nextHour.getTime(),
+        interval: 1000 * 60 * 60,
+        allowImmediate: true,
+      })
+    console.log(`scheduled hourly notify, next notify: ${nextHour.toString()}`)
```
Comment (yukixz) [inline index.es]: remove it

**Comment by yukixz** [comment]
Does it work?

**Comment by KagamiChan** [comment]
用了几天没观察到有问题

**Comment by KagamiChan** [comment]
@yukixz 嗯，用了两天了，没发现什么异常

**Pull request merged**